app/src/test/java/nerd/tuxmobil/fahrplan/congress/alarms/AlarmServicesTest.kt

```diff
@@ -107,21 +106,19 @@ class AlarmServicesTest {
         // AlarmServices invokes discardSessionAlarm() which is tested separately.
         verifyNoInteractions(formattingDelegate)
         verifyInvokedOnce(repository).deleteAlarmForSessionId("S3")
-        assertThat(session.hasAlarm).isFalse()
     }
 
     @Test
-    fun `deleteSessionAlarm resets the session alarm flag when no alarms was scheduled`() {
+    fun `deleteSessionAlarm returns without action when no alarm was scheduled`() {
         val pendingIntentDelegate = mock<PendingIntentDelegate>()
         val formattingDelegate = mock<FormattingDelegate>()
         val alarmServices = createAlarmServices(pendingIntentDelegate = pendingIntentDelegate, formattingDelegate = formattingDelegate)
         whenever(repository.readAlarms(any())) doReturn emptyList()
-        val session = Session("S4").apply { hasAlarm = true }
+        val session = Session("S4") // hasAlarm is not relevant here
```

Comment:
It feels like this comment will be more confusing than helpful when the code is read outside the context of the diff in this pull request. Why mention it in a comment if it's not relevant?

Reply:
:heavy_check_mark: OK. Removed.